book.rb

```diff
@@ -0,0 +1,11 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] The `Book` class should have a method for adding rentals. The method should take in two parameters (`person` and `date`) and return a Rental instance that takes in `date`, `person` and, `self` as parameters.

Reply:
- [ ] The issue here has been properly resolved